postgres-matrix.yml

```diff
@@ -0,0 +1,6 @@
+name: Postgres Version Matrix
+project_name: pg-cron
+
+version_matrix:
+  - 1.4.1:
+      postgres_versions: [10, 11, 12, 13, 14]
```

Comment:
I wonder how we were building pg_cron packages without this file before ?

Reply:
if this file is missing and releasepg is non-empty, we were using releasepg value